### ISSUE TITLE
Referring to letters in notes

### ISSUE THREAD

**Issue by stenskjaer** [lombardpress/lombardpress-schema#29]
If I want to explain something in prose, either by use of a `@type="manual"` on a `<rdg>` element or by a `<note>` (#21) element on a reading, how do I refer to letters?
Let's assume that I can use `<mentioned>` to refer to the words in the text, I have a feeling that you wouldn't use that to refer to letters (= glyphs?).

Take this example:
![skaermbillede 2016-06-28 kl 11 03 31](https://cloud.githubusercontent.com/assets/4047523/16409972/7e946966-3d20-11e6-84d4-d7a985a29cd8.png)

I consider making a note on this like so:

``` xml
Ex illo arguitur sic: Illud non est intelligibile nec scibile per
<app>
  <lem>antecedens</lem>
  <rdg wit="#L">
    <subst>
      <del>consequens</del>
      <add>antecedens</add>
      <note>We have <expan>con</expan><expan>seque</expan>ns with
      <g>a</g> written over the <mentioned>con</mentioned>
      abbreviation.</note>
    </subst>
  </rdg>
</app>
quod non est sensibile, nec per se nec adminus per accidens, sed 
anima intellectiva non
```

So I encode the _a_ as a glyph, but don't really know how to encode the _con_ abbreviation in the note.
Any ideas?

**Comment by stenskjaer** [comment]
Cf. my recent comment on #22, I wonder if maybe, in parallel to that suggestion, it would be better to use the mechanism for encoding linguistic segments like phrases, words, and characters, and to use `<c>` in stead?

**Comment by jeffreycwitt** [comment]
This example is a now a bit out of date, since we don't allow `notes` inside a `rdg` element. Likewise, the use of note here specifically targeted at `#L` should use `witDetail` rather than a `note`.

In any case, in the [present version of the guidelines](https://github.com/lombardpress/lombardpress-schema/blob/ca5b01e/docs/1.0/critical/index.md), we have restricted the contents of `witDetail` to text nodes and and `mentioned` only.

I think the goal was simplicity. Too much complexity places increasing demand on processing. 

I think a complicated and extended description belongs in the general app note. Here we have made such limitation on what can be contained within a note. 

@stenskjaer if you are ok with this for the moment, please close. We can open a new issue with an updated example for the next release.
